lib/vtt.js

```diff
@@ -1311,16 +1311,51 @@
         }
       }
 
-      // 3.2 WebVTT metadata header syntax
-      function parseHeader(input) {
-        parseOptions(input, function (k, v) {
-          switch (k) {
-          case "Region":
-            // 3.3 WebVTT region metadata header syntax
-            parseRegion(v);
+      // draft-pantos-http-live-streaming-20
+      // https://tools.ietf.org/html/draft-pantos-http-live-streaming-20#section-3.5
+      // 3.5 WebVTT
+      function parseTimestampMap(input) {
+        var settings = new Settings();
+
+        parseOptions(input, function(k, v) {
+          switch(k) {
+          case "MPEGT":
+            settings.integer(k + 'S', v);
+            break;
+          case "LOCA":
+            settings.set(k + 'L', parseTimeStamp(v));
             break;
           }
-        }, /:/);
+        }, /[^\d]:/, /,/);
+
+        self.ontimestampmap && self.ontimestampmap({
```

Comment:
so, `ontimestampmap` is a method on the parser that will be called with the MPEGTS and LOCAL times? Does it mean that the user still needs to manually shift all the timestamps by this?